sentry-ruby/lib/sentry/breadcrumb/sentry_logger.rb

```diff
@@ -14,6 +14,7 @@ module SentryLogger
       def add(*args, &block)
         super
         add_breadcrumb(*args, &block)
+        nil
```

Comment:
can you add a test case for this change?

Reply:
Done in rebased-and-squashed f07cb22 =)